js/viz.js

```diff
@@ -325,13 +351,11 @@ const viz = {
 
   dragEnd() {
     d3.event.subject.shadow = false;
-    this.endEvent();
```

Comment:
Also when I move a first party I was kind of expecting the third parties to move closer to where the first party went to also.